packages/meditrak-app/app/entityMenu/helpers.js

```diff
@@ -78,6 +78,10 @@ export const getRecentEntities = (database, state, baseFilters) => {
   return sortedRecentEntities;
 };
 
+/*
+ * This returns a js array, NOT a Realm.Results object
+ * The attributes filter uses a [].filter which converts the results to value iterables
```

Comment:
I originally tried to fix this while maintaining the `Realm.Results` object but that was much more difficult.

`Realm` treats the `attributes` field as a string so we can't compare against the fields within that object. I wasn't able to find a JSON parse within the `Realm` query language so running through `filtered` doesn't work.